src/server/worker/pipeline/transform/transform_test.go

```diff
@@ -257,6 +257,7 @@ func testJobSuccess(t *testing.T, env *testEnv, pi *pps.PipelineInfo, files []ta
 }
 
 func TestTransformPipeline(suite *testing.T) {
+	suite.Skip("TODO: problem with pipeline ids")
```

Comment:
These test are already skipped (by not being run by anything), and they don't work at all.  They all identify pipelines with a "name", but the code demands "name@commit" (and name@master didn't work).

I think the `test-worker-helper` Makefile rule is trying to target them, but misses because it omits "..." at the end:

```
Makefile
314-test-worker-helper:
315-    PROM_PORT=$$(kubectl --namespace=monitoring get svc/prometheus -o json | jq -r .spec.ports[0].nodePort) \
316:      go test -v -count=1 ./src/server/worker/ -timeout $(TIMEOUT) $(TESTFLAGS)
```

These one also missed (but more intentionally):
```
283-test-fuse:
284:    CGOENABLED=0 go test -count=1 -cover $$(go list ./src/server/... | grep '/src/server/pfs/fuse') $(TESTFLAGS)
285-
286-test-local:
287:    CGOENABLED=0 go test -count=1 -cover -short $$(go list ./src/server/... | grep -v '/src/server/pfs/fuse') -timeout $(TIMEOUT) $(TESTFLAGS)

```

```
$ go test -v -count=1 ./src/server/worker/
?       github.com/pachyderm/pachyderm/v2/src/server/worker     [no test files]
```

Reply:
Anyway, I moved the skip to the top of newWorkerSpawnerPair, and fixed the 64 character problem by removing one byte from the name of the failing test.  Also fixed the formatting problem in the ppsdb-generated error.

Reply:
Okay if they never worked, than this shouldn't be a blocker.